src/pattern-library/components/patterns/UtilitiesPage.js

```diff
@@ -70,6 +72,68 @@ export default function UtilitiesPage() {
             </button>
           </Library.Demo>
         </Library.Example>
+        <Library.Example title="Styling groups of inputs">
+          <p>
+            The <code>input-group</code> modifier allows adaptation of styling
+            on inputs when they are part of a group of inputs. This is identical
+            in functionality to the{' '}
+            <Link href="https://tailwindcss.com/docs/hover-focus-and-other-states#styling-based-on-parent-state">
+              tailwind <code>group</code> modifier
+            </Link>{' '}
+            but is {'"namespaced"'} to associate it with its more targeted
+            usage.
+          </p>
+          <Library.Demo title="buttons in an .input-group" withSource>
+            <div className="input-group">
+              <div className="flex gap-x-2 input-group:gap-x-0">
+                <button
+                  aria-expanded
+                  className="border p-2 rounded-lg input-group:rounded-none"
```

Comment:
When I first looked at this I didn't immediately get that the purpose of the `rounded-lg` here was to show it being overridden. I would suggest adding a comment, just for the first button in the group.

Reply:
That's a good suggestion.